### PR TITLE
first version of subvol specification

### PR DESCRIPTION
**Pull request by jreidinger**

### CODE REVIEW

File: crystal/src/y3storage/subvol_specification.cr [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+module Y2Storage
```
Comment (ancorgs) [inline crystal/src/y3storage/subvol_specification.cr]: Y3Storage

File: crystal/src/y3storage/subvol_specification.cr [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+      return true unless arch_specific?
+
+      use_subvol = false
+      archs.each do |a|
+        arch = a.dup
+        negate = arch.start_with?("!")
+        arch[0] = "" if negate # remove leading "!"
+        match = block_given? ? block.call(arch) : arch == target_arch
+        if match && negate
+          log.info("Not using #{self} for explicitly excluded arch #{arch}")
```
Comment (ancorgs) [inline crystal/src/y3storage/subvol_specification.cr]: Oops, I guess these calls to `log.info` should not be here. Not here and not in the Ruby class. Looks like I forgot to remove them.

Comment (jreidinger) [reply crystal/src/y3storage/subvol_specification.cr]: OK, will remove.

**Review by ancorgs — APPROVED** [review]
LGTM

**Pull request merged**